graphgrabber.py

```diff
@@ -47,13 +47,21 @@ def graph_zoom_fit():
     idaapi.process_ui_action('GraphZoomFit')
 
 
-def grab_graph():
-    widget = sark.qt.get_widget('IDA View-A').children()[0]
+def get_ida_view_widget(ida_view_widget_name='IDA View-A'):
```

Comment:
`get_ida_view_widget` -> `get_ida_graph_widget`
We start with the `IDA View-A` widget.